lib/asyn-thread.c

```diff
@@ -163,6 +167,9 @@ struct thread_sync_data {
   char *hostname;        /* hostname to resolve, Curl_async.hostname
                             duplicate */
   int port;
+#ifdef HAVE_SOCKETPAIR
+  int sock_pair[2]; /* socket pair */
```

Comment:
Please use `curl_socket_t` instead of "int" for sockets.

Reply:
done.